solidity_contracts/src/UniswapV2/UniswapV2ERC20.sol

```diff
@@ -87,6 +87,11 @@ contract UniswapV2ERC20 is IUniswapV2ERC20 {
         return true;
     }
 
+    function mint(address to, uint value) external returns (bool) {
```

Comment:
I think that this file is exactly the one in the univ2 library, better update the ERC20.sol imho